rfcs/0123-flake-names.md

```diff
@@ -0,0 +1,89 @@
+---
+feature: flake-names
+start-date: 2022-03-12
+author: Anselm Schüler
+co-authors: None
+shepherd-team: None
+shepherd-leader: None
+related-issues: None
+---
+
+# Summary
+[summary]: #summary
+
+Flakes can declare the field `name`.  
+It represents the name of the flake.  
+The derivations for a flake are no longer called `source`, but use the flake name.
```

Comment:
```suggestion
The store paths corresponding to a flake source are no longer called `source`, but use the flake name.
```

(I’m nitpicking a bit here, but technically these aren’t derivations, they are just paths that are copied directly to the store)

Reply:
Good point!